general/solomiia_tymoshchuk/Task4.java

```diff
@@ -0,0 +1,43 @@
+package general.solomiia_tymoshchuk;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts a day of the week as an integer number (1 - Monday, 2 - Tuesday, …, 7 - Sunday) and output
+ * “Need to go to work…” if the day is a weekday or “Sleeping…”
+ * if it’s a weekend. Create two versions of the program:
+ * one should use the if-else statement, second - the conditional operator.
+ **/
+public class Task4 {
+    String[] weekDaysArray = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"};
+    Integer[] numbers = {1, 2, 3, 4, 5};
+    HashMap<Integer, String> weekDays = new HashMap<Integer, String>();
+
+    public HashMap<Integer, String> hashOfWeekDays() {
```

Comment:
it seems this method is not used

Reply:
@tromanyuk thank you so much for all the comments mentioned. I will check and consider them!